src/ray/raylet/worker_pool_test.cc

```diff
@@ -1348,6 +1406,119 @@ TEST_F(WorkerPoolTest, PopWorkerWithRuntimeEnv) {
   ASSERT_EQ(worker_pool_->GetProcessSize(), 3);
 }
 
+TEST_F(WorkerPoolTest, RuntimeEnvUriReference) {
+  valid_uris.clear();
+  // First part, test URI reference with eager install.
```

Comment:
Can you separate out first / second part to a different `{}` wrapper 

```
{
  /// first part...
}

{
   /// second part
}
```

or a different test suite?

Reply:
Good for me!